examples/main.tf

```diff
@@ -241,3 +241,57 @@ resource "instaclustr_cluster" "example-redis" {
     }
   }
 }
+
```

Comment:
Seeing a couple of issues when trying to plan and apply the following the new examples.

Example - 

`
Error: data.instaclustr_kafka_rest_proxy_user_list.kafka_rest_proxy_user_list: Provider doesn't support data source: instaclustr_kafka_rest_proxy_user_list

Error: data.instaclustr_kafka_schema_registry_user_list.kafka_schema_reg_user_list: Provider doesn't support data source: instaclustr_kafka_schema_registry_user_list
`

Reply:
yeah, I realised that I haven't added these two data sources to the provider.go. Will fix it.

Reply:
Done